src/aof.c

```diff
@@ -1643,6 +1646,10 @@ int loadAppendOnlyFiles(aofManifest *am) {
     server.aof_rewrite_base_size = server.aof_current_size;
     server.aof_fsync_offset = server.aof_current_size;
 
+    /* If we succeeded to load one of the AOF files, and the last AOF file is empty,
+     * the result is AOF_OK (and not AOF_EMPTY) */
+    if (aof_loaded && ret == AOF_EMPTY) ret = AOF_OK;
+
 cleanup:
     stopLoading(ret == AOF_OK);
```

Comment:
i think we need to treat TRUNCATED and maybe EMPTY as success..

Reply:
Agree.

1. if no base and no incr AOF，we just return `AOF_NOT_EXIST`
2. or if `total_size == 0` ， we just return `AOF_EMPTY`
3. if any of the middle AOF is empty ,we convert the ret to `AOF_OK` 
4. if the AOF was truncated (only the last file) , we also treat it as `AOF_OK`

@oranagra right ?

Reply:
yes, that's the logic i aim for.
please check the code to make sure we didn't miss anything.

Reply:
@YaacovHazan  Maybe you need to update your PR to satisfy the above logic. thanks.

Reply:
@chenyang8094 already added a new commit... pls review